tests/unit/controlgroup/controlgroup_options.js

```diff
@@ -0,0 +1,125 @@
+module( "Controlgroup: options" );
+
+function hasCornerClass( className, element ) {
+	if ( className ) {
+		return element.hasClass( className );
+	}
+
+	return element.attr( "class" ).match( /ui-corner/g );
+}
+
+test( "disabled", function() {
+	expect( 1 );
+	var tests = 2,
+		element = $( ".controlgroup" ).controlgroup().controlgroup( "option", "disabled", true );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is present on widget after setting disabled true" );
+	$.each( $.ui.controlgroup.prototype.options.items, function( widget, selector ){
+		element.children( selector ).each(function(){
+			expect( ++tests );
+			strictEqual( $( this )[ widget ]( "widget" ).hasClass( "ui-state-disabled"), true,
+				"Child " + widget + " widgets are disabled" );
+		});
+	});
+	element.controlgroup( "option", "disabled", false );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after setting disabled false" );
+	$.each( $.ui.controlgroup.prototype.options.items, function( widget, selector ){
+		element.children( selector ).each(function(){
+			expect( ++tests );
+			strictEqual( !$( this )[ widget ]( "widget" ).hasClass( "ui-state-disabled"), true,
+				"Child " + widget + " widgets are disabled" );
+		});
+	});
+});
+test( "items", function() {
+	var element = $( ".controlgroup" ).controlgroup({
+		items: {
+			"button": null,
+			"selectmenu": null,
+			"checkboxradio": null
+		}
+	});
+
+	strictEqual( element.children( ".ui-button" ).length, 0,
+		"Child widgets are not called when selector is null" );
+
+	element.controlgroup( "option", "items", {
+		"button": "button"
+	});
+	strictEqual( element.children( ".ui-button" ).length, 2,
+		"Correct child widgets are called when selector is updated" );
+	expect( 2 );
+});
+
+test( "items: custom selector", function() {
+	var element = $( ".controlgroup-custom-selector" ).controlgroup({
+		items: {
+			"button": ".button"
+		}
+	});
+	strictEqual( element.children( ".ui-button" ).length, 5,
+		"Correct child widgets are called when custom selector used" );
+	expect( 1 );
+});
+
+$.widget( "ui.test", {
+	_create: function (){
+		this.element.addClass( "ui-test ui-button" );
+	},
+	refresh: function() {
+		return;
+	}
+});
+test( "items: custom widget", function() {
+	var element = $( ".controlgroup-custom-widget" ).controlgroup({
+		items: {
+			"test": ".test"
+		}
+	});
+
+	strictEqual( element.children( ".ui-button" ).length, 7,
+		"Correct child widgets are called when custom selector used" );
+	strictEqual( element.children( ".ui-test" ).length, 1,
+		"Custom widget called" );
+	expect( 2 );
+});
+
+test( "excludeInvisible", function() {
```

Comment:
Since the default is `true`, asserting that again isn't too useful, should already be covered by the core test (right?). Should start with initializing the option as `false`, then changing it to `true` later and check that the change is handled.